clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1766,13 +1766,14 @@ class SyclKernelIntHeaderCreator : public SyclKernelFieldHandler {
   void addParam(const FieldDecl *FD, QualType ArgTy,
                 SYCLIntegrationHeader::kernel_param_kind_t Kind) {
     uint64_t Size;
+    size_t paramCount = 1;
```

Comment:
No reason for this variable.  However, I WOULD like an assert here to make sure Kind isn't one of the special ones.